src/main/java/play/dev/filewatch/OS.java

```diff
@@ -0,0 +1,22 @@
+package play.dev.filewatch;
+
+import java.util.Locale;
+
+enum OS {
+  Windows,
+  Linux,
+  Mac,
+  Other;
```

Comment:
I moved `OS` into it's own file, so it's a "top-level" citizen in the filewatch package now. You put it in the `FileWatchService` class before, but I think it's nicer if it is standalone. Actually I tried to hide it completely, but because we use it in the tests it's not possible... Would be nice if Java had package access, or interfaces that could contain protected members....